pl.c

```diff
@@ -252,7 +252,7 @@ static void pl_load_one(const char *file)
 	free(full);
 }
 
-static void pl_load_all(void)
```

Comment:
Is this function needed outside of pl.c?

Reply:
Nice catch-- it is *not*. Will fix

Reply:
Please leave `pl_load_all` static.